.goreleaser.yaml

```
@@ -0,0 +1,63 @@
+version: 1
+
+project_name: lava
+
+gomod:
+  proxy: true
+
+builds:
+  - id: lava
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    main: ./cmd/lava
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - "7"
+    ignore:
+      - goos: darwin
+        goarch: arm
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: arm64
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
+        -X main.builtBy=goreleaser
+archives:
+  - format: tar.gz
+    name_template: >-
+      {{ .ProjectName }}_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    format_overrides:
+      - goos: windows
+        format: zip
```

I would use the same arch name used by Go. Also the windows override can go away.

```suggestion
archives:
  - format: tar.gz
    name_template: {{.ProjectName}}_{{.Os}}_{{.Arch}}{{if .Arm}}v{{ .Arm }}{{end}}
```